tests/test_project.py

```diff
@@ -1036,7 +1037,7 @@ def switch_to_cwd(self):
         os.chdir(self.cwd)
 
     @pytest.fixture(autouse=True)
-    def setUp(self,request):
+    def setup_main_interface(self,request,setUp):
         super(TestProjectMainInterface, self).setUp()
```

Comment:
Remove this line 
```suggestion

````

Reply:
Yes I've tried this, it's not working too

Reply:
Okay now I've overcome this error too by no trying to add finalizer in my last line